lesson_13.py

```diff
@@ -0,0 +1,52 @@
+# Task 1
+def func_variables():
+
+    func1 = 123
+    func2 = '123'
+    func3 = [1,2,3]
+    func4 = ('1','2','3')
+    func5 = 6
+
+print(func_variables.__code__.co_nlocals)
+# Task 2
+
+def func_outside(phrase):
+
+    def func_inside(character):
+        print(phrase + character)
+
+    return func_inside
+
+test1 = func_outside('Cheers')
+
+test1('!')
+
+# Task 3
+# Я не розумію, чому мій код не працює ...
+def choose_func(nums, func1, func2):
+    filtration = list(filter(lambda x : True if x > 0 else False, nums)) # а саме оцей фільтер,чомусь
+    # не повертає True False
+    print(filtration)
+    if all(filtration):
+        return func1(nums)
+    elif not all(filtration):
```

Comment:
Redundant elif statement
```suggestion
    return func2(nums)
```

Reply:
I see. Can I ask one question about my code? 
I had some troubles with this part of the program :  filtration = list(filter(lambda x : True if x > 0 else False, nums))
It didn't return True or False.
Does it work when you run?